### PR TITLE
Add table of issues

### PR DESCRIPTION
**Pull request by mosuem**
`flutter run -d chrome` shows the new table, available by switching tabs to `issues` on the top right.

### CODE REVIEW

**Comment by mosuem** [comment]
For some reason, the updating is super slow. don't know why yet...

File: lib/src/issue_table.dart [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+          ),
+          maxLines: 1,
+          overflow: TextOverflow.ellipsis,
+        ),
+      ),
+    );
+  }
+}
+
+ValidationResult? oldPrValidator(Issue issue) {
```
Comment (devoncarew) [inline lib/src/issue_table.dart:197]: We should update this validator to reflect that it's now working on issues (i.e., updating the function name and the validation messages).

File: lib/src/issue_table.dart [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+  final days = DateTime.now().difference(createdAt).inDays;
+  if (days > 365) {
+    return ValidationResult.warning('PR is objectively pretty old');
+  }
+
+  return null;
+}
+
+ValidationResult? needsReviewersValidator(Set<String> googlers, Issue issue) {
```
Comment (devoncarew) [inline lib/src/issue_table.dart:209]: We may want to just remove these validators until we know how we want to validate issues (which ones we want to call out in the UI as being in warning or error states).

File: lib/src/pages/homepage.dart [hunk 3]
```diff
@@ -76,14 +84,32 @@ class _MyHomePageState extends State<MyHomePage> {
               },
             ),
           ),
+          ValueListenableBuilder<DashboardType>(
+            valueListenable: widget.typeSwitch,
+            builder: (BuildContext context, DashboardType value, _) {
+              return DropdownButton<DashboardType>(
```
Comment (devoncarew) [inline lib/src/pages/homepage.dart:90]: Using this looks good; one other UI element to consider is a TabBar + Tabs. They work really well w/ and Scaffold and AppBar - the tabs have an integrated look w/ the AppBar. This is a good example of using them: https://docs.flutter.dev/cookbook/design/tabs.

**Review by devoncarew — APPROVED** [review]
This is great!

> flutter run -d chrome shows the new table

You can also add screenshots to a PR - useful when making a bunch of UI changes.

File: lib/src/pages/homepage.dart (hunk 3 above)
Comment (mosuem) [reply lib/src/pages/homepage.dart:90]: I tried that, can't get it to work somehow. Maybe in a follow-up PR.

**Pull request merged**